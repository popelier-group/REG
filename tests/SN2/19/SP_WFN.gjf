# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.10796700   -0.00006200    0.00018500
 H                 -0.92251300   -0.89934700   -0.56016900
 H                 -0.92247800    0.93487100   -0.49840200
 H                 -0.92259600   -0.03572800    1.05918300
 Br                 1.63552200    0.00000900   -0.00001800
 Cl                -3.23109900    0.00000800   -0.00004100

frame.wfn
