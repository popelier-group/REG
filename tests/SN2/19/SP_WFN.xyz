
6

C       -0.008724    0.093284    0.987124
H       -0.479860   -0.868597    0.887524
H       -0.591782    0.955014    0.714068
H        1.050525    0.141061    0.805961
Br        0.015713   -0.164884   -1.744082
Cl       -0.028029    0.293162    3.100739
