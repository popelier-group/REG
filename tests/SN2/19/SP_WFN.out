 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.19/Gau-28185.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.19/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     28186.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.10797  -0.00006   0.00019 
 H                    -0.92251  -0.89935  -0.56017 
 H                    -0.92248   0.93487  -0.4984 
 H                    -0.9226   -0.03573   1.05918 
 Br                    1.63552   0.00001  -0.00002 
 Cl                   -3.2311    0.00001  -0.00004 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.107967   -0.000062    0.000185
      2          1           0       -0.922513   -0.899347   -0.560169
      3          1           0       -0.922478    0.934871   -0.498402
      4          1           0       -0.922596   -0.035728    1.059183
      5         35           0        1.635522    0.000009   -0.000018
      6         17           0       -3.231099    0.000008   -0.000041
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.075687   0.000000
     3  H    1.075683   1.835258   0.000000
     4  H    1.075691   1.835249   1.835248   0.000000
     5  Br   2.743489   2.768782   2.768703   2.768962   0.000000
     6  Cl   2.123132   2.540109   2.540091   2.540161   4.866621
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.008724    0.093284    0.987124
      2          1           0       -0.479860   -0.868597    0.887524
      3          1           0       -0.591782    0.955014    0.714068
      4          1           0        1.050525    0.141061    0.805961
      5         35           0        0.015713   -0.164884   -1.744082
      6         17           0       -0.028029    0.293162    3.100739
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         148.8815381           0.8491231           0.8491231
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.0389642129 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  4.33D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39283869     A.U. after   12 cycles
            NFock= 12  Conv=0.28D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.57995-101.36767 -62.23311 -56.05128 -56.05063
 Alpha  occ. eigenvalues --  -56.05063 -10.08874  -9.28072  -8.44198  -7.04332
 Alpha  occ. eigenvalues --   -7.03672  -7.03672  -6.26632  -6.26405  -6.26405
 Alpha  occ. eigenvalues --   -2.37846  -2.37782  -2.37782  -2.37603  -2.37603
 Alpha  occ. eigenvalues --   -0.64519  -0.55126  -0.50743  -0.30536  -0.30536
 Alpha  occ. eigenvalues --   -0.21217  -0.14752  -0.14752  -0.07578  -0.07438
 Alpha  occ. eigenvalues --   -0.07438
 Alpha virt. eigenvalues --    0.10501   0.11496   0.15208   0.15208   0.16773
 Alpha virt. eigenvalues --    0.17188   0.19205   0.19205   0.21015   0.21080
 Alpha virt. eigenvalues --    0.21081   0.22687   0.22688   0.24756   0.29218
 Alpha virt. eigenvalues --    0.31507   0.31508   0.37222   0.39580   0.54371
 Alpha virt. eigenvalues --    0.54371   0.56258   0.57070   0.59166   0.59166
 Alpha virt. eigenvalues --    0.62189   0.62189   0.69006   0.77201   0.77201
 Alpha virt. eigenvalues --    0.80867   0.81202   0.85762   0.85762   0.86497
 Alpha virt. eigenvalues --    0.86497   0.93843   1.03463   1.03464   1.05136
 Alpha virt. eigenvalues --    1.05136   1.08342   1.16944   1.23726   1.23727
 Alpha virt. eigenvalues --    1.26111   1.53909   1.64366   1.64367   1.76537
 Alpha virt. eigenvalues --    1.92478   1.92478   1.99222   2.17621   2.17622
 Alpha virt. eigenvalues --    2.24168   2.24169   2.32110   2.44639   2.44640
 Alpha virt. eigenvalues --    2.53268   2.62764   2.62764   2.66008   2.74413
 Alpha virt. eigenvalues --    2.74413   3.07819   3.19606   3.85711   3.85712
 Alpha virt. eigenvalues --    4.37432   4.37432   4.38315   4.38316   4.55715
 Alpha virt. eigenvalues --    6.76764   7.78207   7.78212   7.85571   9.84366
 Alpha virt. eigenvalues --   24.03831  26.00111  26.00112  26.36307  48.07650
 Alpha virt. eigenvalues --  215.82519 289.98969 289.98970 290.061721020.84063
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.308566   0.419592   0.419596   0.419585  -0.021843  -0.094727
     2  H    0.419592   0.513414  -0.035487  -0.035498  -0.015643  -0.033981
     3  H    0.419596  -0.035487   0.513409  -0.035495  -0.015650  -0.033979
     4  H    0.419585  -0.035498  -0.035495   0.513424  -0.015626  -0.033982
     5  Br  -0.021843  -0.015643  -0.015650  -0.015626  35.837879   0.036005
     6  Cl  -0.094727  -0.033981  -0.033979  -0.033982   0.036005  17.467575
 Mulliken charges:
               1
     1  C   -0.450770
     2  H    0.187603
     3  H    0.187607
     4  H    0.187592
     5  Br  -0.805122
     6  Cl  -0.306911
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.112033
     5  Br  -0.805122
     6  Cl  -0.306911
 Electronic spatial extent (au):  <R**2>=           1138.1260
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.0068    Y=              0.0752    Z=              0.7967  Tot=              0.8003
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -44.1191   YY=            -44.4391   ZZ=            -80.2496
   XY=              0.0309   XZ=              0.3265   YZ=             -3.4163
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             12.1501   YY=             11.8302   ZZ=            -23.9803
   XY=              0.0309   XZ=              0.3265   YZ=             -3.4163
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.3325  YYY=             -3.5488  ZZZ=            -86.5280  XYY=             -0.8965
  XXY=             -0.9181  XXZ=            -11.7083  XZZ=              0.5608  YZZ=             -5.9691
  YYZ=            -12.1467  XYZ=              0.1407
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -73.5653 YYYY=            -85.3292 ZZZZ=          -1601.6807 XXXY=              0.6282
 XXXZ=              6.6823 YYYX=              0.3679 YYYZ=            -62.8712 ZZZX=              7.7986
 ZZZY=            -81.9153 XXYY=            -26.4303 XXZZ=           -245.1254 YYZZ=           -248.8677
 XXYZ=            -20.7280 YYXZ=              1.3164 ZZXY=              0.4976
 N-N= 1.710389642129D+02 E-N=-7.695280834269D+03  KE= 3.069977108901D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE525\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.107967,-0.000062,0.000185\H,0,-0.922513,-0.899347,-0.560169\H,0
 ,-0.922478,0.934871,-0.498402\H,0,-0.922596,-0.035728,1.059183\Br,0,1.
 635522,0.000009,-0.000018\Cl,0,-3.231099,0.000008,-0.000041\\Version=E
 S64L-G16RevC.01\State=1-A\HF=-3074.3928387\RMSD=2.808e-09\Dipole=-0.31
 48539,-0.0000531,0.0001627\Quadrupole=-18.0711225,9.0355975,9.035525,0
 .0000662,-0.0003262,-0.0000003\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 AND ALL THIS SCIENCE I DON'T UNDERSTAND
 IT'S JUST MY JOB FIVE DAYS A WEEK

                           -- ELTON JOHN, "ROCKET MAN"
 Job cpu time:       0 days  0 hours  0 minutes 27.5 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:49:53 2020.
